tests/test_control_panel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from unittest import mock
+
+from tests import IntegrationTestCase
+from tests.fixtures import UserFactory
+
+
+@mock.patch('dashboard.service.fetch_user_data')
+class TestControlPanel(IntegrationTestCase):
+    def test_user_cannot_see_control_panel_in_navbar(self, fetch_data):
```

Comment:
Please add test for case, when user is not log in. For now in this case application makes ka-buuuuum